modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -23,9 +27,9 @@ You can use a Try scope in each route of a Scatter-Gather component to handle an
 
 To illustrate how this works, consider the following two cases:
 
-* The routes in a Scatter-Gather component each contain a Try scope. One of the routes generates an error that is successfully handled by that route’s Try scope, so the route is still able to generate a Mule event. The Scatter-Gather component consolidates the Mule events from all routes into a new Mule event, as usual, and passes the consolidated event to the next event processor.
+* The routes in a Scatter-Gather component each contain a Try scope. One of the routes generates an error that is successfully handled by that route’s Try scope through an `on-error-continue`, so the route is still able to generate a Mule event. The Scatter-Gather component consolidates the Mule events from all routes into a new Mule event and passes the consolidated event to the next event processor.
 
-* An error occurs in a route. This route either does not contain a Try scope or the error is of a type that the Try scope cannot handle. Although all the other routes complete processing, the failed route causes the Scatter-Gather component to throw a `MULE.COMPOSITE ERROR` error, which the flow passes to the Error Handler component that you have configured for your flow. The error-handling event processors are able to process the Mule events from the completed routes.
+* An error occurs in a route. This route either does not contain a Try scope or the error is of a type that the Try scope cannot handle. Although all the other routes complete processing, the failed route causes the Scatter-Gather component to throw a `MULE:COMPOSITE_ROUTING` error, which the flow passes to the Error Handler component that you have configured for your flow. The error-handling event processors are able to process the Mule events from the completed routes.
```

Comment:
`or the error is of a type that the Try scope cannot handle` -> `or the error is of a type that the Try scope doesn't have a handler for, or the handler is an "on-error-propagate"`